.github/workflows/build_main_documentation.yml

```diff
@@ -9,6 +9,8 @@ on:
       package:
         required: true
         type: string
+      package_name:
```

Comment:
Would it make sense to add a `description` field to help us / other doc-builder users understand what this arg does? See the [docs](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#using-inputs-and-outputs-with-an-action) for an idea of what I mean.